cfdemos/droneflybetween.py

```diff
@@ -71,24 +71,20 @@ def run_sequence(scf):
     Flies to the midpoint of the two controller's
     """
     cf = scf.cf
-    with PositionHlCommander(
-                scf,
-                default_velocity=2,
-                default_height=0.5) as pc:
-        while True:
-                    
-            poses = vr.getDeviceToAbsoluteTrackingPose(
-                openvr.TrackingUniverseStanding, 0,
-                openvr.k_unMaxTrackedDeviceCount)
-
-            pos1 = get_tracker_pos(tracker1)
-            pos2 = get_tracker_pos(tracker2)
-            setpoint = [(pos1[0] + pos2[0]) / 2,(pos1[1] + pos2[1]) / 2,(pos1[2] + pos2[2]) / 2]
-            
-            pc.go_to(setpoint[0],
-                     setpoint[1],
-                     setpoint[2])
-        pc.go_to(0.0, 0.0, 0.0)
+    height_above_center = 0.1
+    while True:
+                
+        poses = vr.getDeviceToAbsoluteTrackingPose(
+            openvr.TrackingUniverseStanding, 0,
+            openvr.k_unMaxTrackedDeviceCount)
+
+        pos1 = get_tracker_pos(tracker1)
+        pos2 = get_tracker_pos(tracker2)
+        setpoint = [(pos1[0] + pos2[0]) / 2,(pos1[1] + pos2[1]) / 2,(pos1[2] + pos2[2]) / 2 + height_above_center]
```

Comment:
Avoid large unnamed formulas
```suggestion
        midpointX = (pos1[0] + pos2[0]) / 2
        midpointY = (pos1[1] + pos2[1]) / 2
        midpointZ = (pos1[2] + pos2[2]) / 2
        setpoint = [midpointX, midpointY, midpointZ + height_above_center]
```